packages/stores/types.ts

```diff
@@ -535,6 +535,11 @@ export type TCoreStores = {
     modules: Record<string, any>;
     notifications: TNotificationStore;
     traders_hub: TTradersHubStore;
+    gtm: Record<string, unknown>;
+    pushwoosh: Record<string, unknown>;
+    contract_replay: Record<string, unknown>;
+    chart_barrier_store: Record<string, unknown>;
+    active_symbols: Record<string, unknown>;
```

Comment:
@maryia-deriv I recommend not doing this, We should either type things properly or if out of the scope of this PR just leave it as it was with the TS error for the next person to fix 🙇🏻